example/pubspec.yaml

```diff
@@ -27,5 +27,17 @@ dev_dependencies:
     sdk: flutter
   flutter_lints: ^1.0.0
 
+dependency_overrides:
+  combined_barcode_scanner:
+    path: ../
+  fast_barcode_scanner:
+    git:
+      url: git://github.com/ikbendewilliam/fast_barcode_scanner.git
```

Comment:
this will not be publishable to pub.dev 😢

Reply:
I know :-(